README.md

```diff
@@ -50,7 +50,7 @@ Then assign it to your target:
 The plugin can then be used within any publishing pipeline like this:
 
 ```swift
-import PublishReadingTime
+import PublishReadingTimePlugin
```

Comment:
It's `ReadingTimePublishPlugin`

Reply:
I think I fixed it. Is there anything else you need?